src/Candidate.js

```diff
@@ -46,3 +60,35 @@ export const CandidateName = styled.span`
 export const CandidateVoteCounts = styled.span`
```

Comment:
의미상 단수가 더 맞겠지?

우리말로 `개수 + 들` 대신 `개수`를 쓰는 것처럼 여기서도 `counts`보단 `count`가 적합할 것 같아.

```suggestion
export const CandidateVoteCount = styled.span`
```